Test/allocated1/dafny0/Computations.dfy

```diff
@@ -1,3 +1,3 @@
-// RUN: %dafny /verifyAllModules /allocated:1 /compile:0 /print:"%t.print" /dprint:"%t.dprint" "%s" > "%t"
+// RUN: %dafny /verifyAllModules /allocated:1 /proverOpt:O:smt.qi.eager_threshold=25 /compile:0 /print:"%t.print" /dprint:"%t.dprint" "%s" > "%t"
```

Comment:
Is this an option we should (right now) suggest as a verification technique in the manual and (slightly later) add a Dafny-level option for of some kind?

Reply:
I've considered that. I'd like to experiment with it a bit more on production code and see if it makes any difference there.